src/libguac/user-handlers.c

```diff
@@ -676,6 +677,21 @@ int __guac_handshake_image_handler(guac_user* user, int argc, char** argv) {
     
 }
 
+int __guac_handshake_name_handler(guac_user* user, int argc, char** argv) {
+
+    /* Free any past value */
+    free((char *) user->info.name);
+
+    if (argc > 0 && strcmp(argv[0], ""))
```

Comment:
I think this would benefit from a comment describing what this is for, and maybe some more descriptive variable names.

Reply:
Okay, I've added some additional comments to that area, and I renamed the `name` member of the `info` struct to `real_name`. I was having trouble coming up with anything that fit it any better - as usual, open to suggestions. I need to update the client-side code, as well.

Reply:
Actually, scratch that, I went with "username". I can do "client_username" if that makes more sense....